### ISSUE TITLE
How do I install this?

### ISSUE THREAD

**Issue by digital-raven** [fufexan/dotfiles#28]
Hey guys, this may be a newbie question, but I'm confused how to install your package. I thought your screen shots were really impressive and I love the look, but I'm confused how to install your software.

Your README mentions `nix run github:fufexan/dotfiles#packageName` but then I get an error output about `experimental Nix feature 'nix-command' is disabled; add --extra-experimental-features nix-command' to enable it.

Could you add a "typical" example to your README to get from basic hyprland to your setup. I don't know what I'm supposed to point at and I don't want to have any awkward changes to my machine when I'm done.

**Comment by fufexan** [comment]
> Your README mentions `nix run github:fufexan/dotfiles#packageName` but then I get an error

Yeah, that's a general format for the command that should be run with the packages in `pkgs/`. The configs aren't listed there though.



> Could you add a "typical" example to your README to get from basic hyprland to your setup. I don't know what I'm supposed to point at and I don't want to have any awkward changes to my machine when I'm done.

Generally it's either "copy everything to the right place", or directly install the `io` machine, after changing the `hardware-configuration.nix` file in its directory.
